apps/frontend/src/app/5_pages/ErrorPage/ErrorPage.tsx

```diff
@@ -33,7 +34,7 @@ export const ErrorPage: FC = () => {
             <Paragraph className="w-full p-3">
               <i>{error.statusText || error.message}</i>
             </Paragraph>
-            {process.env.NODE_ENV === 'development' && (
+            {isDevEnvironment() && (
```

Comment:
Pls update 2 other locations where we use the condition (`wdyr.ts` and `useDepositSocket`) as well.